app/views/job_runs/_structural_updates_summary.html.erb

```diff
@@ -0,0 +1,25 @@
+<div class="structural_changes">
+    <h5>Updates from csv</h5>
```

Comment:
Given a11y concerns around heading levels, and the inside-a-table context, I wonder if considering the `<caption>` element (instead of `<h?>`) is suitable here. If so, it may require some CSS to get it displaying per the design.

Reply:
This is matching the existing structure, but I should probably go ahead and edit those areas to improve accessibility as well. :+1:

Reply:
Good call 🙂